bitcoin/src/pow.rs

```diff
@@ -2034,7 +2038,6 @@ mod tests {
     #[test]
     fn u256_to_f64() {
         // Validate that the Target::MAX value matches the constant also used in difficulty calculation.
```

Comment:
You forgot to remove the comment attached to the line you deleted.

Reply:
Thanks for pointing it.